build.py

```diff
@@ -322,7 +322,7 @@ def gen_generator(self, language):
     self.do_compile(params, target_filename, filenames, remove)
 
   def do_compile(self, params, target_filename, filenames, remove):
-    do_compile = self.do_compile_remote if self.closure_env == REMOTE_COMPILER else self.do_compile_local
+    do_compile = self.do_compile_remote if self.closure_env["closure_compiler"] == REMOTE_COMPILER else self.do_compile_local
```

Comment:
I believe this has since been resolved with https://github.com/LLK/scratch-blocks/pull/1512